conf/broker.conf

```diff
@@ -1283,16 +1284,12 @@ splitTopicAndPartitionLabelInPrometheus=false
 # The schema storage implementation used by this broker
 schemaRegistryStorageClassName=org.apache.pulsar.broker.service.schema.BookkeeperSchemaStorageFactory
 
-# Enforce schema validation on following cases:
-#
-# - if a producer without a schema attempts to produce to a topic with schema, the producer will be
-#   failed to connect. PLEASE be carefully on using this, since non-java clients don't support schema.
-#   if you enable this setting, it will cause non-java clients failed to produce.
+# Whether to enable schema validation.
+# When enabled, if a producer without a schema attempts to produce to a topic with schema, the producer will be rejected and disconnected.
```

Comment:
```suggestion
# When schema validation is enabled, if a producer without a schema attempts to produce to a topic with a schema, the producer is rejected and disconnected.
```

1. or "produce to a topic with schemas"?

2. Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. Details: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

Reply:
> 1. or "produce to a topic with schemas"?

"The producer sends a message without schema to the topic that has one or many schemas"?

> 2. Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. Details: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

Thanks